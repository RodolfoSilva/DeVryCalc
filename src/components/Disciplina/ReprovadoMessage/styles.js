import { StyleSheet } from 'react-native'

export default StyleSheet.create({
  container: {
    alignItems: 'center',
    paddingVertical: 26
  },
  message: {
    fontSize: 14,
    fontWeight: '500',
    color: '#333'
  }
})
