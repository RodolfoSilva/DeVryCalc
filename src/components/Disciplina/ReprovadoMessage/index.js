import React from 'react'
import { Text, View } from 'react-native'
import styles from './styles'

const ReprovacaoMessage = () => (
  <View style={styles.container}>
    <Text style={styles.message}>
      Não foi desssa vez, sorte na próxima....
    </Text>
  </View>
)

export default ReprovacaoMessage
