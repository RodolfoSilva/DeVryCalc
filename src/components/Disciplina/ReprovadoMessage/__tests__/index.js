import React from 'react'
import ReprovadoMessage from '../'
import { shallow } from 'enzyme'

describe('ReprovadoMessage', () => {
  it('Renderiza a mensagem de reprovação', () => {
    const wrapper = shallow(<ReprovadoMessage />)
    expect(wrapper).toMatchSnapshot()
  })
})
