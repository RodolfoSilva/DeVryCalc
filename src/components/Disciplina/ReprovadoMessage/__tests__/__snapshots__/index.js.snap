// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReprovadoMessage Renderiza a mensagem de reprovação 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "paddingVertical": 26,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333",
        "fontSize": 14,
        "fontWeight": "500",
      }
    }
  >
    Não foi desssa vez, sorte na próxima....
  </Text>
</View>
`;
